src/monster.cpp

```diff
@@ -3371,7 +3371,10 @@ void monster::hear_sound( const tripoint &source, const int vol, const int dist,
     if( wander_turns < wandf ) {
         return;
     }
-    process_trigger( mon_trigger::SOUND, volume );
+    // only trigger this if the monster is not friendly or the source isn't the player
+    if( friendly >= 0 || source != get_player_character().pos() ) {
```

Comment:
If the comment is correct, then:
```suggestion
    // only trigger this if the monster is not friendly or the source isn't the player
    if( friendly == 0 || source != get_player_character().pos() ) {
```